third_party/amd/backend/compiler.py

```diff
@@ -157,6 +157,7 @@ def make_llir(src, metadata, options, capability):
         pm.run(mod)
         pm = ir.pass_manager(mod.context)
```

Comment:
I don't get why we create multiple pass_managers in here, cleanup ?

Reply:
yes, let's keep it for later :)